setenv.sh

```diff
@@ -0,0 +1 @@
+export LD_LIBRARY_PATH=${PWD}/external/mkl/mklml_lnx_2017.0.0.20160801/lib:$LD_LIBRARY_PATH
```

Comment:
write a readme and instruct the user to add this to their .bashrc.

Overall I think it's better to make mkl installation separate from mxnet.
say you have a install_mklml.sh that downloads, compiles, and installs mkl to PREFIX and ask the user to append LD_LIBRARY_PATH=PREFIX/lib to their bashrc.

Some users like to install mxnet into their python site-packages instead of using it inplace